src/fake-timers-src.js

```diff
@@ -143,6 +155,30 @@ function withGlobal(_global) {
         throw new TypeError("now should be milliseconds since UNIX epoch");
     }
 
+    function getInfiniteLoopError(loopLimit, job) {
+        var infiniteLoopError = new Error(
+            "Aborting after running " +
+                loopLimit +
+                " timers, assuming an infinite loop!"
+        );
+
+        var sliceSize = job.type ? 4 : 3;
+
+        infiniteLoopError.stack =
```

Comment:
Some environments don't have a writeable stack property - so it would be better to wrap this in a try/catch and to use defineProperty

Reply:
In particular we had this issue several times in bluebird which also mutates stacks